src/js/Drawer/Drawer.js

```diff
@@ -31,10 +41,28 @@ class DrawerSide extends Component {
     this.setState({ drawerIsOpen: isOpen });
   };
 
+  handleCloseNotification = () => {
+    this.setState({ notification: false });
+  };
+
   render() {
-    const { drawerIsOpen } = this.state;
+    const { drawerIsOpen, notification } = this.state;
+    const notificationText = 'Error. Please contact system administrator';
```

Comment:
This will be redefined every time component renders. Make it class property e.g.